src/plugin/debugger/mhpdbgc.c

```diff
@@ -1022,7 +1022,12 @@ static void mhp_dump_to_file(int argc, char *argv[])
 
   fd = open(argv[3], O_WRONLY | O_CREAT | O_TRUNC | O_CLOEXEC, 00640);
   if (fd < 0) {
-    mhp_printf("cannot open/create file %s\n%s\n", argv[3], strerror(errno));
+    int e = errno;
+    mhp_printf("cannot open/create file %s\n", argv[3]);
+    if (e == EACCES)
+      mhp_printf("perhaps try running dosemu with '-no-priv-sep'\n");
```

Comment:
Do we need this dump_to_file
stuff? And `-no-priv-sep`...
Debugger already has a few
`B_printf`'s, so what about
using those and just remove this?

Reply:
I think dumping to a file could be useful, but happy to drop if you want to?

Reply:
Unless you know the reason
for otherwise, dumping to a
log is perfectly fine with me.
And it won't need to disable
anything (like landlock).

Also it may be useful to only
dump to the log, as its much,
much faster that to terminal
via pipes. So it would be cool
if `tc` is extended with the
`silent` mode that dumps
only to the log.

Reply:
Dump to log sounds okay.

Reply:
So how about silent mode
that dumps _only_ to the
log (and not to terminal)?

Reply:
Oh yes that sounds fine.

Reply:
I added 'memory dump to log and 'tc silent' to my list of things to do.